### PR TITLE
#162163259 Integrate coverage service

### PR DESCRIPTION
**Pull request by joshtrigger**
**What does this PR do?**
This Pull Request integrates Coveralls code coverage service with the code base.

**Description**
The coverage service will help inform us on how much of the code is being covered by the tests.

**What are the relevant pivotal tracker stories?**
#162163259

### CODE REVIEW

**Comment by KicaRonaldOkello** [comment]
i think it is ready to merge

**Review by SokoPaulSokool — APPROVED** [review]

**Review by KicaRonaldOkello — APPROVED** [review]

**Review by bibangamba — APPROVED** [review]

File: authors/settings.py [hunk 1]
```diff
@@ -78,6 +78,18 @@
 # Database
 # https://docs.djangoproject.com/en/1.11/ref/settings/#databases
 
+if 'TRAVIS' in os.environ:
+    DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': 'athenatraviscidb',
+        'USER': 'postgres',
```
Comment (collin5) [inline authors/settings.py]: Please pick these from the environment. https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings

**Review by collin5 — CHANGES_REQUESTED** [review]

File: authors/settings.py (hunk 1 above)
Comment (joshtrigger) [reply authors/settings.py]: Thanks for the feedback, let me get to it immediately.

**Review by collin5 — CHANGES_REQUESTED** [review]
LGTM. Please resolve conflicts.

**Review by Kasulejoseph — APPROVED** [review]

**Review by kafuuma — APPROVED** [review]

**Comment by joshtrigger** [comment]
collins I implemented the feedback

**Review by collin5 — APPROVED** [review]

**Pull request merged**